### PR TITLE
fix: trigger handleGetStarted only on welcome screen

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-5217

### CODE REVIEW

**Review by chybisov — APPROVED** [review]

File: packages/dapp/src/components/Widgets/Widgets.tsx [hunk 1]
```diff
@@ -84,6 +84,8 @@ export function Widgets() {
   }, [activeTab, setActiveTab, starterVariant, starterVariantUsed]);
 
   const handleGetStarted: MouseEventHandler<HTMLDivElement> = (event) => {
+    if (welcomeScreenEntered) return;
```
Comment (Addminus) [inline packages/dapp/src/components/Widgets/Widgets.tsx:87]: Can we rename this property? it is confunsing. i would read it as he entered IN the welcome screen (he opened it)

**Review by Addminus — CHANGES_REQUESTED** [review]

File: packages/dapp/src/components/Widgets/Widgets.tsx (hunk 1 above)
Comment (Addminus) [reply packages/dapp/src/components/Widgets/Widgets.tsx:87]: welcome screen closed

Comment (dennyscode) [reply packages/dapp/src/components/Widgets/Widgets.tsx:87]: Note: Change "welcomeScreenEntered" --> use welcomeScreenClosed

**Comment by dennyscode** [comment]
Latest changes:
- renamed zustand-state to `welcomeScreenClosed`
- only pass onClick event if welcome screen has NOT been closed, yet

**Pull request merged**